frontend/public/components/dashboard/dashboards-page/cluster-dashboard/details-card.tsx

```diff
@@ -42,6 +42,12 @@ const ClusterVersion: React.FC<ClusterVersionProps> = ({ cv }) => {
   const desiredVersion = getDesiredClusterVersion(cv);
   const lastVersion = getLastCompletedUpdate(cv);
   const status = getClusterUpdateStatus(cv);
+  const clusterVersionIsEditable = useAccessReview({
+    group: ClusterVersionModel.apiGroup,
+    resource: ClusterVersionModel.plural,
+    verb: 'patch',
+    name: cv?.metadata?.name,
```

Comment:
We might hard-code this as `version` to avoid firing off multiple self-subject access review requests if it hasn't loaded loaded.

```suggestion
    name: 'version',
```